ogr/ogrsf_frmts/hana/ogrhanautils.cpp

```diff
@@ -27,12 +27,25 @@
  ****************************************************************************/
 
 #include "ogrhanautils.h"
+#include "cpl_string.h"
 
 #include <algorithm>
 
 namespace OGRHANA
 {
 
+HANAVersion HANAVersion::fromVersionString(const char *version)
```

Comment:
The 'Version' string seems to be redundant in the function name.
HANAVersion::fromString(...)?

Reply:
Done.